UpdatedImageProcessing/TargetDetection/color_operations.py

```diff
@@ -0,0 +1,120 @@
+import cv2
+import math
+import numpy
+import PIL.ImageOps
+from PIL import Image
+from sklearn.cluster import MiniBatchKMeans
+
+class ColorOperations(object):
+
+    @staticmethod
+    def find_distance(color_1, color_2):
+        return math.sqrt(((color_1[0] - color_2[0]) ** 2) + ((color_1[1] - color_2[1]) ** 2) + ((color_1[2] - color_2[2]) ** 2))
+
+    @staticmethod
+    def find_percentage_difference(color_1, color_2):
+        color_distance = ColorOperations.find_distance(color_1, color_2)
+        return (color_distance / math.sqrt(((255) ** 2) + ((255) ** 2) + ((255) ** 2))) * 100
+
+    @staticmethod
+    def quantize_color(rgb_image, kmeans_clusters_number):
+        (h, w) = (rgb_image.height, rgb_image.width)
```

Comment:
Is there any particular reason you are using these variables? Since you are not editing them, it makes sense to just use the ```rgb_image.height``` and ```rgb_image.width``` variables because it reduces code complexity.